docs/connector-development/connector-builder-ui/maintenance.md

```diff
@@ -0,0 +1,175 @@
+
+Goal: Ensuring the changes work as expected and provide a safety net for the next modifications
+
+Expected output: CATs are green
+
+To ensure we can validate the changes on a connector, CATs are executed. A Connector Acceptance Test is a piece of software that emulates a worker (see architecture [here](../../understanding-airbyte/jobs.md)) and checks the output of commands that the connector is expected to implement. In order to configure these tests, we need to:  
+* Create tests scenarios in a sandbox environment: There is no streamlined process to do this. Each product has their own business rules that can't be captured in a unifying document. Given a new connector, this process might require to have pay for a subscription for the product in order to get credentials/workspace/etc... In order to be able to repopulate an API very easily, we suggest the use of a Postman collection or a script.
+* Configure what we want to be validated: Using `acceptance-test-config.yml`, you can toggle some of the validations that are performed. The most common cases are:
+    * If `expected_records.jsonl` to validate the records that the source returns has been added, ensure `acceptance_tests.basic_read.tests.expect_records` is populated. This might require you to create a catalog for this test suites.
+    * If a stream was made incremental, ensure `acceptance_tests.incremental.tests.expect_records` is populated. This might require you to create a catalog for this test suites.
+* Set up the configuration: In order to run properly, CATs require a valid config. Those configs often rely on secret. You can use [GSM](https://console.cloud.google.com/security/secret-manager?project=dataline-integration-testing) to modify the existing configuration. Simply write the connector name in the filter bar, click on `SECRET_SOURCE-<name>__CREDS` and `NEW VERSION` and upload the new config.
+
+To execute CATs, there are two options.
+
+### Locally
+We recommend using `cd airbyte-integrations/connections/source-<name> && ./acceptance-test-docker.sh` to run them using docker. You might have to change the permissions to run this file. Once it's done, the results will be printed to the terminal.
+
+### In a GitHub pull request
+Given that you have pushed you changes on a branch in airbytehq/airbyte, go on https://github.com/airbytehq/airbyte and you should see a yellow banner saying "<branch> had recent pushes less than a minute ago". Click on "Compare & pull request", fill out the template and click on "Create pull request". A check named "Connectors tests / Connectors CI" will run. Once it's done, a new comment will be added to show the test results. 
+
+## Release
+
+Goal: Make the connector available to end-users
+
+Expected output: https://hub.docker.com/r/airbyte/source-<name>/tags shows a recently created tag
+
+The following steps assume that you are getting
+
+* Bump version (Docker file and metadata.yaml) and update changelogs under `docs/integrations`
```

Comment:
Isn't this and the breaking changes playbook part of the checklist? Do we need to mention it here?